python/cuml/experimental/fil/fil.pyx

```diff
@@ -294,6 +295,23 @@ cdef class ForestInference_impl():
             output_dtype=output_dtype
         )
 
+
+class _AutoIterations:
+    """Used to generate sequence of iterations (1, 2, 5, 10, 20, 50...) during
+    FIL optimization"""
+    sequence = (1, 2, 5)
```

Comment:
I think I understand the reasoning for using a class variable here, but in practice this often leads to hard-to-find bugs, should the sequence be changed after all for a specific instance. Would it be ok to just instantiate that on init with default?

Reply:
Yep, I'm fine with that. Will update accordingly.